docs/html/development/issue-triage.rst

```diff
@@ -0,0 +1,305 @@
+
+**trivial**
+  special label that when used on a PR removes the news file requirement
+
+**needs rebase or merge**
+  this is a special label used by BrownTruck to mark PRs that have merge
+  conflicts
+
+
+Automation
+----------
+
+There are several helpers to manage issues and pull requests.
+
+Issues created on the issue tracker are automatically given the
+``triage`` label by the
+`triage-new-issues <https://github.com/apps/triage-new-issues>`__
+bot. The label is automatically removed when another label is added.
+
+When an issue needs feedback from the author we can label it with
+``S: awaiting response``. When the author responds, the
+`no-response <https://github.com/apps/no-response>`__ bot removes the label.
+
+After an issue has been closed for 30 days, the
+`lock <https://github.com/apps/lock>`__ bot locks the issue and adds the
+``S: auto-locked`` label. This allows us to avoid monitoring existing closed
+issues, but unfortunately prevents and references to issues from showing up as
+links on the closed issue.
+
+
+Triage Issues
+*************
+
+Users can make issues for a number of reasons:
+
+#. Suggestions about pip features that could be added or improved
+#. Problems using pip
+#. Concerns about pip usability
+#. General packaging problems to be solved with pip
+#. Problems installing or using Python packages
+#. Problems managing virtual environments
+#. Problems managing Python installations
+
+To triage issues means to identify what kind of issue is happening and
+
+* confirm bugs
+* provide support
+* discuss and design around the uses of the tool
+
+Specifically, to address an issue:
+
+#. Read issue title
+#. Scan issue description
+#. Ask questions
+#. If time is available, try to reproduce
+#. Search for or remember related issues and link to them
+#. Identify an appropriate area of concern (if applicable)
+
+Keep in mind that all communication is happening with other people and
+should be done with respect per the
+`Code of Conduct <https://www.pypa.io/en/latest/code-of-conduct/>`__.
+
+The lifecycle of an issue (bug or support) generally looks like:
+
+#. waiting for triage (marked with label ``triage``)
+#. confirming issue - some discussion with the user, gathering
+   details, trying to reproduce the issue (may be marked with a specific
+   category, ``S: awaiting-respose``, ``S: discussion-needed``, or
+   ``S: need-repro``)
+#. confirmed - the issue is pretty consistently reproducible in a
+   straightforward way, or a mechanism that could be causing the issue has been
+   identified
+#. awaiting fix - the fix is identified and no real discussion on the issue
+   is needed, should be marked ``R: awaiting PR``
+#. closed - can be for several reasons
+
+   * fixed
+   * could not be reproduced, no more details could be obtained, and no
+     progress can be made
+   * actual issue was with another project or related to system
+     configuration that pip cannot improve its behavior in the context
+     of
```

Comment:
It looks it is missing some words ?

Reply:
Reworded to be more clear [here](https://github.com/pypa/pip/pull/6868/files#diff-0ee19f321d04652294c191edc037e9a7R146-R147).